localoam/index.js

```diff
@@ -0,0 +1,52 @@
+var http = require('http');
+var filed = require('filed');
+var glob = require('glob');
+var fs = require('fs');
+
+var master = {
+  'nodes': [
+    {
+      'locations': [
+        {
+          'type': 'localoam',
+          'url': 'http://localoam:4999'
```

Comment:
Does it make sense to make this configurable so that the app can easily be run outside of a docker-compose context?